content/support-matrix/delfin/huawei/oceanstor/_index.md

```diff
@@ -0,0 +1,517 @@
+         <td>Type</td>
+         <td>Type of port</td>
+      </tr>
+      <tr>
+         <td>Logical type</td>
+         <td>Port logical type. Ex:frontend, maintenance, service, management, etc.</td>
+      </tr>
+      <tr>
+         <td>Speed</td>
+         <td>Speed of the ports</td>
+      </tr>
+      <tr>
+         <td>Max Speed</td>
+         <td>Maximum speed of the ports</td>
+      </tr>
+      <tr>
+         <td>Parent Id</td>
+         <td>Name of controller</td>
+      </tr>
+      <tr>
+         <td>WWN</td>
+         <td>WWN</td>
+      </tr>
+      <tr>
+         <td>mac address</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td>ipv4</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td>ipv4 mask</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td>ipv6</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td>ipv6 mask</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td rowspan=17>Disk</td>
+         <td>Name</td>
+         <td>Name of storage resource</td>
+      </tr>
+      <tr>
+         <td>Storage Id</td>
+         <td>It is storage id within delfin system</td>
+      </tr>
+      <tr>
+         <td>Disk id</td>
+         <td>Specifies the disk ID.</td>
+      </tr>
+      <tr>
+         <td>Serial number</td>
+         <td>Serial Number</td>
+      </tr>
+      <tr>
+         <td>Manufacturer</td>
+         <td>Manufacturer</td>
+      </tr>
+      <tr>
+         <td>Model</td>
+         <td>Model</td>
+      </tr>
+      <tr>
+         <td>Firmware</td>
+         <td>Firmware Version</td>
+      </tr>
+      <tr>
+         <td>Speed</td>
+         <td>Speed of the port</td>
+      </tr>
+      <tr>
+         <td>Capacity</td>
+         <td>It is measured in terms of bytes</td>
+      </tr>
+      <tr>
+         <td>Status</td>
+         <td>Status</td>
+      </tr>
+      <tr>
+         <td>Physical type</td>
+         <td>Disk physical type. Ex:sata, sas, ssd, etc.</td>
+      </tr>
+      <tr>
+         <td>Logical type</td>
+         <td>Disk logical type. Ex:frontend, maintenance, service, management, etc. </td>
+      </tr>
+      <tr>
+         <td>Health Score</td>
+         <td>Disk health score</td>
+      </tr>
+      <tr>
+         <td>Disk group id</td>
+         <td></td>
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td></td>
+      </tr>
+   </tbody>
+</table>
+
+### Alert data matrix
+
+<table>
+   <thead>
+      <tr>
+         <th>Attribute</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Alert id</td>
+         <td>Identification of alarm</td>
+      </tr>
+      <tr>
+         <td>Alert name</td>
+         <td>Name of the alert</td>
+      </tr>
+      <tr>
+         <td>Severity</td>
+         <td>Severity of the alert</td>
+      </tr>
+      <tr>
+         <td>Category</td>
+         <td>Type of the reported alarm notification. Ex: Fault, Event, Recovery</td>
```

Comment:
Category of the reported alarm notification. Ex: Fault, Event, Recovery

Reply:
done